index.html

```diff
@@ -3640,8 +3451,7 @@ <h3><dfn>Writing or pushing content</dfn></h3>
         <ol>
           <li>
             If |reader|.<a>[[\Id]]</a> is [= dictionary member/present =]
-            and doesn't <a href="#dfn-match-record-identifier-with-url-pattern">match</a>
-            any |record:NDEFRecord|'s id where |record|
+            and it is not equal to any |record|'s id where |record|
```

Comment:
Fair enough, an id should be compared for equality, not for partial match. Good.